modules/vaos/vaos.gemspec

```diff
@@ -6,15 +6,23 @@ $LOAD_PATH.push File.expand_path('lib', __dir__)
 require 'vaos/version'
 
 # Describe your gem and declare its dependencies:
-Gem::Specification.new do |spec|
-  spec.name        = 'vaos'
-  spec.version     = VAOS::VERSION
-  spec.authors     = ['Alastair Dawson']
-  spec.email       = ['alastair.j.dawson@gmail.com']
-  spec.homepage    = 'https://api.va.gov'
-  spec.summary     = 'VAOS API'
-  spec.description = 'VOAS (VA Online Scheduling) is part of VAMF (VA Mobile Framework) and allows veterans to make and manage appointments for care'
-  spec.license     = 'MIT'
+Gem::Specification.new do |s|
```

Comment:
why change this variable to `s`?

Reply:
It was to match the way the other modules do it.